### ISSUE TITLE
Server config gets ignored. 

### ISSUE THREAD

**Issue by Aaron995** [AmyMialeeMods/no-enchant-cap#8]
Hey 

The config I have on my server gets ignored by clients making them able to edit their config and ignoring whatever config the server has.

**Comment by AmyMialee** [comment]
Does this actually let them do anything? Like isn't all the enchantment stuff handled server side anyway?

**Comment by Aaron995** [comment]
Yes I was able to get protection on an axe while the server didn't allow that. Once I switched that option off on my local client i wasn't able to do it.
